semgrep_output_v1.atd

```diff
@@ -89,14 +89,21 @@ type location
 }
 
 (*****************************************************************************)
-(* Rule id *)
+(* Rules *)
 (*****************************************************************************)
 
 (* e.g., "javascript.security.do-not-use-eval" *)
 type rule_id
   <ocaml attr="deriving show">
   <python decorator="dataclass(frozen=True)"> = string
 
+type rule_id_and_engine_kind
+  <ocaml attr="deriving show">
+  <python decorator="dataclass(frozen=True)"> = {
+  rule_id: rule_id;
```

Comment:
You can use a pair instead of a record. it's a bit ugly to see rule_id: rule_id; engine_kind: engine_kind;